iroh/src/node/builder.rs

```diff
@@ -231,9 +231,9 @@ fn mk_external_rpc() -> IrohServerEndpoint {
 impl Default for Builder<iroh_blobs::store::mem::Store> {
     fn default() -> Self {
         // Use staging in testing
-        #[cfg(not(any(test, feature = "test-utils")))]
+        #[cfg(not(all(test, feature = "test-utils")))]
```

Comment:
This should probably be just `cfg(not(test))`?  If someone is requesting `test-utils` they can still enable `test` themselves.  Unless this breaks one of the benchmarks again...